netman/adapters/memory_session_storage.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2016 Internap.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from netman.core.objects.exceptions import SessionAlreadyExists, UnknownSession
+from netman.core.session_storage import SessionStorage
+
+
+class MemorySessionStorage(SessionStorage):
+    def __init__(self):
+        super(MemorySessionStorage, self).__init__()
+        self.sessions = {}
+
+    def add(self, switch_descriptor, session_id):
```

Comment:
wouldn't it make more sense that session id be the first param?

i mean imagine this scales to greater responsibilities and you now need to add a switch_color variable...

def add(self, switch_descriptor, session_id, switch_color=None, ....):

putting the session ID first, all subsequent values are "part of the session identified by this id"